content/blog/2022-07-01-Galaxy-Wellbeing-Blog-2/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Defining Key Concepts in Galaxy Well-being and Mental Health Project"
+tease: "Second blog post presenting the Outreachy project on Mental Health"
+authors: "Melkeberhan Berhanu Degefa, Beatriz Serrano-Solano, Anne Fouilloux"
+source_blog: "Galaxy Well-Being Outreachy project"
+date: "2022-07-01"
+---
+
+In our last blog post, we wrote a brief introduction to what the [Galaxy Well-being and Mental Health project](https://galaxyproject.org/blog/2022-06-29-galaxy-wellbeing-project/) is and what it entails. Now let’s break down the key terms we are using in this project.
+
+# What is Well-being?
+
+**Well-being** can be defined as the overall sense of health characterized by positive functioning. It has two aspects which are subjective and objective well-being that crucially recognises parts of our lives that we determine ourselves through our potential, how we feel about ourselves, how we form meaningful relationships and our sense of purpose as well as aspects such as economy, health, education and so on. It is considered to constitute 8 main dimensions which are emotional, social, occupational, physical, spiritual, environmental, intellectual and financial. If any of these dimensions are neglected over time, it will be detrimental to our well-being.
+
+# What is Mental Health?
+
+**Mental health**, according to WHO, can be understood as a state of well-being in which an individual recognizes his or her abilities or potential, can cope with everyday stressors, can work effectively, and contribute to his or her community. Mental health is critical to our individual and collective abilities to think, feel, communicate with one another, and enjoy life. Mental health promotion can be viewed as a critical issue for people, communities, and societies all over the world. Mental health problems can affect anyone and can manifest in a variety of ways. Thus, fostering positive mental health benefits everyone. It is not a one-off process but a continuous work which requires consistency and patience.
```

Comment:
```suggestion
**Mental health**, according to [WHO](https://www.who.int/), can be understood as a state of well-being in which an individual recognizes his or her abilities or potential, can cope with everyday stressors, can work effectively, and contribute to his or her community. 
```

Reply:
Is the whole paragraph the definition? If not, I'd split it into 2, or put the definition quoted/italics.

Reply:
Yes its the whole paragraph. I didn't put in quote/italics because there are some changed words. I will edit it.